mathquill4quill.js

```diff
@@ -394,7 +433,7 @@ if (typeof module !== "undefined" && module.exports) {
 
 // for backwards compatibility with prototype-based API
 if (window.Quill) {
-  window.Quill.prototype.enableMathQuillFormulaAuthoring = function(options) {
+  window.Quill.prototype.enableMathQuillFormulaAuthoring = function (options) {
```

Comment:
Nit: let's remove this diff to keep the history clean and focused to the minimal required changes for this feature.
```suggestion
  window.Quill.prototype.enableMathQuillFormulaAuthoring = function(options) {
```